src/server/stages/repository/strapiStages.repository.test.ts

```diff
@@ -0,0 +1,153 @@
+			jest.spyOn(errorManagementService, 'handleFailureError').mockReturnValue(expectedFailure);
+			const slug = 'slug';
+			const offreDeStageAvecDonnéesErronées = anOffreDeStageResponse({ domaines: 35 } as unknown as Partial<OffreStageResponseStrapi.OffreStage>);
+			jest.spyOn(strapiService, 'getFirstFromCollectionType').mockResolvedValue(createSuccess(offreDeStageAvecDonnéesErronées));
+			const strapiStagesRepository = new StrapiStagesRepository(strapiService, errorManagementService);
+
+			const stage = await strapiStagesRepository.getOffreDeStageBySlug(slug);
+
+			expect(stage).toStrictEqual(expectedFailure);
+			expect(errorManagementService.handleFailureError).toHaveBeenCalledWith(expect.any(Error), aLogInformation({
+				apiSource: 'STRAPI - offre de stage',
+				contexte: 'get offre de stage par slug',
+				message: 'impossible de mapper les stages par slug',
+			}));
+		});
+
+		it('lorsque la requete est en echec, renvoie cet echec', async () => {
+			const slug = 'slug';
+			const strapiService = aStrapiCmsRepository();
+			const expectedFailure = createFailure(ErreurMetier.CONTENU_INDISPONIBLE);
+			jest.spyOn(strapiService, 'getFirstFromCollectionType').mockResolvedValueOnce(expectedFailure);
+
+			const strapiStagesRepository = new StrapiStagesRepository(strapiService, anErrorManagementService());
+			const stage = await strapiStagesRepository.getOffreDeStageBySlug(slug);
+
+			expect(stage).toStrictEqual(expectedFailure);
+		});
+	});
+
+	describe('listAllOffreDeStageSlug', () => {
+		it('appelle le strapi service avec les bons paramètres et renvoie un success de l‘offre de stage mappé', async () => {
+			const strapiService = aStrapiCmsRepository();
+			jest.spyOn(strapiService, 'getCollectionType').mockResolvedValueOnce(createSuccess([anOffreDeStageResponse({ slug: 'slug 1' }), anOffreDeStageResponse({ slug: 'slug 2' })]));
+			const query = 'fields[0]=slug';
+
+			const strapiStagesRepository = new StrapiStagesRepository(strapiService, anErrorManagementService());
+			await strapiStagesRepository.listAllOffreDeStageSlug();
+
+			expect(strapiService.getCollectionType).toHaveBeenCalledTimes(1);
+			expect(strapiService.getCollectionType).toHaveBeenCalledWith(RESOURCE_OFFRE_DE_STAGE, query);
+		});
+
+		it('lorsque la requete est en succès, renvoie un success avec la liste des slugs', async () => {
+			const strapiService = aStrapiCmsRepository();
+			jest.spyOn(strapiService, 'getCollectionType').mockResolvedValueOnce(createSuccess([anOffreDeStageResponse({ slug: 'slug 1' }), anOffreDeStageResponse({ slug: 'slug 2' })]));
+
+			const strapiStagesRepository = new StrapiStagesRepository(strapiService, anErrorManagementService());
+			const listSlugs = await strapiStagesRepository.listAllOffreDeStageSlug();
+
+			expect(listSlugs).toStrictEqual(createSuccess(['slug 1', 'slug 2']));
+		});
+
+		it('lorsque la requete est en echec, renvoie cet echec', async () => {
```

Comment:
```suggestion
		it('lorsque la requête est en échec, renvoie cet échec', async () => {
```
et sur les autres occurrences dans les noms de test